tests/dummy/app/templates/index.hbs

```diff
@@ -45,6 +45,11 @@
   <p>You can find out more information on the CSP addon page <a href="https://github.com/rwjblue/ember-cli-content-security-policy#ember-cli-content-security-policy">here</a>.</p>
   <p>We also need to allow `data:` in `img-src` because of a <a href="https://github.com/miguelcobain/ember-paper/blob/master/app%2Fstyles%2Fpaper-button.scss">current hack</a> in paper-button.</p>
 
+  <p>
+    Finally, the examples contain a `transition-to` helper that is not installed by default. To use this helper run the following:
```

Comment:
Howdy @DanChadwick - `ember install ember-route-action` is just fine. I'll move it to the sidenav when I'm back down there (today sometime)